src/Controller/QueryFieldRestController.php

```diff
@@ -81,4 +108,20 @@ private function getContentType(ContentInfo $contentInfo): ContentType
 
         return $contentTypes[$contentInfo->contentTypeId];
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return array
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    private function loadLocationByPath(Request $request): Location
```

Comment:
`array`/`Location`? Which is it :smile: ?

Reply:
Fixed.